public/ticketmanager.php

```diff
@@ -542,10 +543,24 @@
                     echo "</span>";
                     echo "</div>";
                     echo "</td>";
-
                     echo "</tr>";
                 }
 
+                echo "<tr>";
+                echo "<td></td><td colspan='6'>";
+
+                $userAwarded = getUserUnlockAchievement( $reportedBy, $achID, $unlockData );
+                if( $userAwarded )
+                {
+                    echo "$reportedBy earned this achievement at ". getNiceDate( strtotime( $unlockData[0][ 'Date' ] ) );
```

Comment:
@luchaos I'm under the impression that hardcoding the `0` index here looks a bit "inelegant". Am I being too nitpicky here?